tests/e2e/tests/pilot/pilot_test.go

```diff
@@ -46,18 +47,20 @@ const (
 
 var (
 	tc = &testConfig{
-		V1alpha1: true, //implies envoyv1
-		V1alpha3: true, //implies envoyv2
-		Ingress:  true,
-		Egress:   true,
+		V1alpha1:    true, //implies envoyv1
+		V1alpha3:    true, //implies envoyv2
```

Comment:
That's the default config ? Can you change it to only V1alpha3 ( flags can override ) - there is no case where we want to test both. I would turn egress off by default too.